in_toto/common_args.py

```diff
@@ -223,6 +223,20 @@ def parse_password_and_prompt_args(args):
     ),
 }
 
+INSPECT_TIMEOUT_ARGS = ["--inspection-timeout"]
+INSPECT_TIMEOUT_KWARGS = {
+    "type": int,
+    "dest": "inspect_timeout",
+    "default": LINK_CMD_EXEC_TIMEOUT,
+    "help": (
+        "integer that represents the max timeout in seconds for the "
+        "   in-toto-verify command for inspect subprocess."
+        "   Default is '{timeout}' seconds.".format(
+            timeout=LINK_CMD_EXEC_TIMEOUT
+        )
+    ),
+}
+
```

Comment:
IMO you can move this directly to `in_toto_verify.py`, it doesn't look like it will be re-used by another command (same is true for RUN_TIMEOUT, but we don't have to fix that here).